### DISCUSSION TITLE
Trim Reblog Options

### DISCUSSION THREAD

**Discussion by Medstromx13** [AprilSylph/XKit-Rewritten#263] [Help Desk]
I'm getting use to the new features. With the trim reblogs feature, which is awesome so far, will there be an option to save the last to previous posts. Sometimes with RP, if it's a three person thread, you'll want to keep at least the last two posts along with yours. Just something I was curious about.

Thank you for all your hard work!

**Comment by AprilSylph** [comment]
I can see myself implementing that, yes! The current version is just the bare minimum of what I could build; from the start, I wanted the feature to give more control over how the post was trimmed.
